src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -207,14 +237,15 @@ private String attachCaseByLegacyId(AttachToCaseEventData event) {
                 "Found case with CCD ID '{}' for legacy ID '{}' (attaching exception record '{}')",
                 targetCaseCcdId,
                 event.targetCaseRef,
-                event.exceptionRecordId
+                event.exceptionRecord.getId()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Potential violation of Law of Demeter (static property access)](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4558409)